src/js/components/DataTable/README.md

```diff
@@ -0,0 +1,116 @@
+## DataTable
+A data driven table.
+
+[![](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=datatable&module=%2Fsrc%2FDataTable.js)
+## Usage
+
+```javascript
+import { DataTable } from 'grommet';
+<DataTable />
+```
+
+## Properties
+
+**bodyProps**
+
+Properties to pass to configure body cells.
+
+```
+{
+
```

Comment:
why this is an empty json object ?

Reply:
Oops. i forgot to run `generate-readme` before committing. Will do.